docs/chaos/index.md

```diff
@@ -30,19 +30,54 @@ builder
 // Finally, configure chaos strategies if you want to inject chaos.
 // These should come after the regular resilience strategies.
 
-// 2% of invocations will be injected with chaos
-const double InjectionRate = 0.02;
+// 2% of all requests will be injected with chaos fault.
+const double faultInjectionRate = 0.02;
+// For the remaining 98% of total requests, 50% of them will be injected with latency. Then 49% of total request will be injected with chaos latency.
+// Latency injection does not return early.
+const double latencyInjectionRate = 0.50;
+// For the remaining 98% of total requests, 10% of them will be injected with outcome. Then 9.8% of total request will be injected with chaos outcome.
+const double outcomeInjectionRate = 0.10;
+// For the remaining 88.2% of total requests, 1% of them will be injected with behavior. Then 0.882% of total request will be injected with chaos behavior.
+const double behaviorInjectionRate = 0.01;
 
 builder
-    .AddChaosLatency(InjectionRate, TimeSpan.FromMinutes(1)) // Inject a chaos latency to executions
-    .AddChaosFault(InjectionRate, () => new InvalidOperationException("Injected by chaos strategy!")) // Inject a chaos fault to executions
-    .AddChaosOutcome(InjectionRate, () => new HttpResponseMessage(System.Net.HttpStatusCode.InternalServerError)) // Inject a chaos outcome to executions
-    .AddChaosBehavior(0.001, cancellationToken => RestartRedisAsync(cancellationToken)); // Inject a chaos behavior to executions
+    .AddChaosFault(faultInjectionRate, () => new InvalidOperationException("Injected by chaos strategy!")) // Inject a chaos fault to executions
+    .AddChaosLatency(latencyInjectionRate, TimeSpan.FromMinutes(1)) // Inject a chaos latency to executions
+    .AddChaosOutcome(outcomeInjectionRate, () => new HttpResponseMessage(System.Net.HttpStatusCode.InternalServerError)) // Inject a chaos outcome to executions
+    .AddChaosBehavior(behaviorInjectionRate, cancellationToken => RestartRedisAsync(cancellationToken)); // Inject a chaos behavior to executions
 ```
 <!-- endSnippet -->
 
 > [!NOTE]
```

Comment:
This note is now broken:

![image](https://github.com/user-attachments/assets/973ec05d-9b95-4e5d-afcc-e60566188580)